tests/NuGet.Jobs.Catalog2Registration.Tests/Schema/EntityBuilderFacts.cs

```diff
@@ -221,6 +221,89 @@ public void PopulatesProperties()
 }",
                     json);
             }
+
+            [Fact]
+            public void PopulatesVulnerabilityProperties()
+            {
+                Hive = HiveType.SemVer2;
+                var leaf = V3Data.Leaf;
+                leaf.Vulnerabilities = new List<PackageVulnerability>() {
+                    new PackageVulnerability
+                    {
+                        Id = "https://apidev.nugetfoo.org/v3/catalog0/data/2020.07.06.06.49.47/bar.1.0.0.json#vulnerability/GitHub/999",
```

Comment:
nit: avoid domains that may be owned by someone else. apidev.nugettest.org is fine, or just https://example